src/Microsoft.OData.Client/ClientEdmModel.cs

```diff
@@ -21,13 +21,7 @@ namespace Microsoft.OData.Client
     using Microsoft.OData.Edm;
     using Microsoft.OData.Edm.Vocabularies;
     using c = Microsoft.OData.Client;
-
-#if PORTABLELIB

```

Comment:
@tchuan, just had a discussion with the team and we think the appropriate change is actually to make the preprocessor target specifically Windows Phone 8.0 instead of removing the preprocessor entirely. This is due to the fact that we still build a test project against Windows Phone 8.0 (using OData Client portable lib, non-Net Standard version) and even if we don't plan to ship the portable lib, we don't want to publish a broken test until we've removed the test project and related projects (which we currently don't have the resources to review at this time).

Therefore, I recommend changing the preprocessor directive from
`#if PORTABLELIB`
to
`#if PORTABLELIB && WINDOWSPHONE`

The `WINDOWSPHONE` constant is defined in the [Windows Phone test project](https://github.com/OData/odata.net/blob/master/test/EndToEndTests/Tests/Client/Build.PortableLibrary.Phone/Microsoft.Test.OData.Tests.Client.Phone.csproj#L18) that I referred to.